phabfive/cli.py

```diff
@@ -180,6 +187,37 @@ def run(cli_args, sub_args):
                     print(created_uri)
                 elif sub_args["list"]:
                     d.print_uri(repo=sub_args["<repo>"])
+                elif sub_args["edit"]:
+                    object_id = d.get_object_identifier(
+                        repo_name=sub_args["<repo>"], uri_name=sub_args["<uri>"]
+                    )
+                    if sub_args["--enable"]:
+                        disable = False
+                    elif sub_args["--disable"]:
+                        disable = True
+                    else:
+                        disable = None
+
+                    if (
+                        sub_args["--new_uri"] is None  # noqa: W503
+                        and sub_args["--io"] is None  # noqa: W503
+                        and sub_args["--display"] is None  # noqa: W503
+                        and sub_args["--cred"] is None  # noqa: W503
+                        and disable is None  # noqa: W503
```

Comment:
Try `if not any([sub_args["this"], sub_args["that"], sub_args["etc"]])`.

`any()` and `all()` for condition testing is a good thing to know and use.

Reply:
`not` checks for False or True. The if-statement should check for `None` because the value of `disable` is `True`, `False` or `None`.

Using `all()`, iterating over the list and check for `None`